src/main/java/uk/gov/ons/census/casesvc/messaging/DeactivateUacReceiver.java

```diff
@@ -0,0 +1,50 @@
+package uk.gov.ons.census.casesvc.messaging;
+
+import static uk.gov.ons.census.casesvc.utility.JsonHelper.convertObjectToJson;
+import static uk.gov.ons.census.casesvc.utility.MsgDateHelper.getMsgTimeStamp;
+
+import java.time.OffsetDateTime;
+import org.springframework.integration.annotation.MessageEndpoint;
+import org.springframework.integration.annotation.ServiceActivator;
+import org.springframework.messaging.Message;
+import org.springframework.transaction.annotation.Transactional;
+import uk.gov.ons.census.casesvc.logging.EventLogger;
+import uk.gov.ons.census.casesvc.model.dto.ResponseManagementEvent;
+import uk.gov.ons.census.casesvc.model.dto.UacDTO;
+import uk.gov.ons.census.casesvc.model.entity.EventType;
+import uk.gov.ons.census.casesvc.model.entity.UacQidLink;
+import uk.gov.ons.census.casesvc.service.UacService;
+
+@MessageEndpoint
+public class DeactivateUacReceiver {
+
+  private static final String DEACTIVATE_UAC = "DEACTIVATED UAC";
```

Comment:
Shouldn't be ALL CAPS if we're going to match the format of other descriptions, for consistency